src/main/java/org/apache/accumulo/testing/ingest/TestIngest.java

```diff
@@ -226,11 +227,8 @@ public static void ingest(AccumuloClient client, FileSystem fs, Opts opts,
     FileSKVWriter writer = null;
 
     if (opts.outputFile != null) {
-      writer = FileOperations
-          .getInstance()
-          .newWriterBuilder()
-          .forFile(opts.outputFile + "." + RFile.EXTENSION, fs, conf,
-              CryptoServiceFactory.newDefaultInstance())
+      writer = FileOperations.getInstance().newWriterBuilder()
+          .forFile(opts.outputFile + ".rf", fs, conf, CryptoServiceInstance())
```

Comment:
These changes are better but I think this could be done using just the RFile user API.  Checkout [RFile](https://accumulo.apache.org/1.9/apidocs/org/apache/accumulo/core/client/rfile/RFile.html#newWriter--) and [RFileWriter](https://accumulo.apache.org/1.9/apidocs/org/apache/accumulo/core/client/rfile/RFileWriter.html).  This would eliminate the need to provide a CryptoService.

Reply:
Sorry Keith beat me to the punch!